src/opentime/rationalTime.cpp

```diff
@@ -94,6 +94,69 @@ is_dropframe_rate(double rate)
     return std::find(b, e, rate) != e;
 }
 
+static bool
+parseFloat(char const* pCurr, char const* pEnd, bool allow_negative, double* result)
+{
+    if (pCurr >= pEnd || !pCurr) {
+        *result = 0.0;
+        return false;
+    }
+
+    double ret = 0.0;
+    double sign = 1.0;
+
+    if (*pCurr == '+')
+        ++pCurr;
```

Comment:
Do we have a code convention/opinion on mixing braces or non-braces?

Reply:
the rest of the file is as bracey as possible, and new lines before braces. I'll update

Reply:
There's a `.clang-format` file in the root of the repo.  Running clang-format should confirm everything to the same standard.